treeherder/etl/jobs.py

```diff
@@ -278,26 +277,6 @@ def _load_job(repository, job_datum, push_id):
         push_id=push_id,
     )
 
-    artifacts = job_datum.get('artifacts', [])
-
-    has_text_log_summary = any(x for x in artifacts if x['name'] == 'text_log_summary')
```

Comment:
text_log_summary and performance_data are being processed by the log parser

Reply:
This must have been lots of work but what joy to see all of this go away!

Reply:
Yeah, it was confusing having all of this cruft here. Hopefully it'll be easier for the next person needing to work on this part of the codebase!